package.json

```diff
@@ -35,14 +35,15 @@
     "ufo": "^1.5.3"
   },
   "peerDependencies": {
-    "next-auth": "~4.21.1"
+    "next-auth": "5.0.0-beta.19"
```

Comment:
why do we still require next-auth and not auth/...?

Reply:
Great catch, I will try to replace this with `@auth/core` (and update the docs)